packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -56,15 +58,18 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
     private tooltipElement: TimeGraphStateComponent | undefined;
     private tooltipInfo: { [key: string]: string } | undefined;
 
-    private onSelectionChanged = (payload: { [key: string]: string; }) => this.doHandleSelectionChangedSignal(payload);
+    private annotationMarkers: string[] = [];
+    private onSelectionChanged = ( payload: { [key: string]: string; } ) => this.doHandleSelectionChangedSignal(payload);
+    private onMarkerFilterChanged = ( annotationMarkers: string[]) => this.doHandleMarkerFilterChangedSignal(annotationMarkers);
 
     constructor(props: TimegraphOutputProps) {
         super(props);
         this.state = {
             outputStatus: ResponseStatus.RUNNING,
             timegraphTree: [],
             collapsedNodes: [],
-            columns: []
+            columns: [],
+            annotationMarkers:[]
```

Comment:
After a restart with trace and view already opened, there is no marker filter changed signal received, so the list of requested markers is empty, even though the filter shows all selected.

(this is a problem only after fixing not omitting requested_marker_categories parameter when the list is empty)

Reply:
I am not seeing this issue. @bhufmann could you please confirm??

Reply:
@ankusht-work Not sure if it's the same issue, but I notice some inconsistency between the checked state in the menu and what is shown: 
1) Open a trace trace-1 and view with a marker category category-A
2) Deselect cat-A, view hides category-A
3) Open a second trace trace-2 and view with marker category category-A.  Menu shows category-A checked but view doesn't show markers of category-A

Reply:
This has been fixed in the latest uploaded version [5cf9d08](https://github.com/theia-ide/theia-trace-extension/commit/5cf9d08716013d5b879c7b1c787791f42859f31c).